sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -9,15 +9,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
-using Azure.Core.Pipeline;
 using Azure.Messaging.ServiceBus.Core;
 using Azure.Messaging.ServiceBus.Diagnostics;
 
 namespace Azure.Messaging.ServiceBus
 {
     /// <summary>
     ///   A client responsible for sending <see cref="ServiceBusMessage" /> to a specific Service Bus entity
-    ///   (Queue or Topic). It is constructed by calling <see cref="ServiceBusClient.CreateSender(string)"/>.
+    ///   (Queue or Topic). It can be used for session enabled entities. It is constructed by calling <see cref="ServiceBusClient.CreateSender(string)"/>.
```

Comment:
can we say that it can be used for both session and non-session entities? Not sure if we can say "non-session", but just to make it clear somehow that this is used for all sending regardless of the entity.